pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -66763,6 +66763,25 @@ self: {
        license = lib.licenses.bsd2;
      }) {};
 
+  "constraints_0_12" = callPackage
+    ({ mkDerivation, base, binary, deepseq, ghc-prim, hashable, hspec
+     , hspec-discover, mtl, semigroups, transformers, transformers-compat
+     , type-equality
+     }:
+     mkDerivation {
+       pname = "constraints";
+       version = "0.12";
+       sha256 = "f26b531a15f013f15b4b5df46c4f89b13927253a4195494c50320adf05760223";
+       libraryHaskellDepends = [
+         base binary deepseq ghc-prim hashable mtl semigroups transformers
+         transformers-compat type-equality
+       ];
+       testHaskellDepends = [ base hspec ];
+       testToolDepends = [ hspec-discover ];
+       description = "Constraint manipulation";
+       license = lib.licenses.bsd2;
+     }) {};
```

Comment:
You can generate this by adding it to:

https://github.com/NixOS/nixpkgs/blob/599582499c548c8ad7f95dc0b2cb4d276795abeb/pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml#L97

and running `./maintainers/scripts/haskell/regenerate-hackage-packages.sh`